### DISCUSSION TITLE
Research hide/lock/privatise an account/profile so as to remove it from public view

### DISCUSSION THREAD

**Discussion by ldexterldesign** [mastodon/mastodon#30075] [Q&A] [answered]
👋

Hope you're OK!

Is there a way to hide/lock/privatise an account/profile so as to remove it from public view temporarily - sorry, bit unsure of what search terms to use and surprised to not find much in the discussions/issues here after 10 minutes..? 😕

If no then I'll create a feature request

Yours hopefully

**Comment by ldexterldesign** [comment]
BUMP

**Comment by trwnh** [comment, marked as answer]
https://github.com/mastodon/mastodon/issues/7132
